src/web/rustdoc.rs

```diff
@@ -1599,6 +1599,22 @@ mod test {
         })
     }
 
+    #[test]
+    fn test_redirect_to_latest_301() {
+        wrapper(|env| {
+            env.fake_release().name("dummy").version("1.0.0").create()?;
+            let web = env.frontend();
+            let client = ClientBuilder::new()
+                .redirect(redirect::Policy::none())
+                .build()
+                .unwrap();
+            let url = format!("http://{}/dummy", web.server_addr());
+            let status = client.get(url).send()?.status();
+            assert_eq!(status, StatusCode::MOVED_PERMANENTLY);
```

Comment:
can we also test that the target is `/dummy/latest/dummy`?

Reply:
Note that I was not able to use `assert_redirect` because that does not assert a specific status code. It seemed small enough that duplicating a couple of lines was better than adding an "expected status" parameter an updating all tests.